tests/product/test_control.py

```diff
@@ -58,35 +58,35 @@ def assert_service_fails_without_presto(self, service):
         self.assertEqualIgnoringOrder(presto_not_installed,
                                       '\n'.join(start_output))
 
-    def test_server_start_various_states(self):
+    def test_server_start_coordinator_started(self):
         self.setup_cluster(NoHadoopBareImageProvider(),
                            self.STANDALONE_PRESTO_CLUSTER)
 
-        # Coordinator started, workers not; then server start
-        process_per_host = \
-            self.assert_start_with_one_host_started(
-                self.cluster.internal_master)
+        self.assert_start_with_one_host_started(self.cluster.internal_master)
 
-        # Worker started, coord and other workers not; then server start
-        self.run_prestoadmin('server stop').splitlines()
-        self.assert_stopped(process_per_host)
+    def test_server_start_worker_started(self):
+        self.setup_cluster(NoHadoopBareImageProvider(),
+                           self.STANDALONE_PRESTO_CLUSTER)
         self.assert_start_with_one_host_started(
```

Comment:
The point for this one was that start works if one host is already started; you should add a start all after this